<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seveneves</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLPanel" name="GLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>731</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QFrame" name="parameterFrame">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>-220</y>
      <width>441</width>
      <height>731</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="simOptionsBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>431</width>
       <height>471</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>500</height>
      </size>
     </property>
     <property name="title">
      <string>Simulation Options</string>
     </property>
     <widget class="QGroupBox" name="activeForcesBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>431</width>
        <height>230</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>230</height>
       </size>
      </property>
      <property name="title">
       <string>Active Forces</string>
      </property>
      <widget class="QCheckBox" name="gravityCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>97</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Gravity</string>
       </property>
      </widget>
      <widget class="QLabel" name="springLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Spring Stiffness</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="springEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="shearLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Shear Stiffness:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="shearEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="bendLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bend Stiffness:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bendEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>90</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="dampLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Damp Stiffness:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dampEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="collisionRepulsionCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Collision Repulsion</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="stretchingCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Stretching</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="shearingCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Shearing</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="bendingCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bending</string>
       </property>
      </widget>
      <widget class="QLabel" name="clothWidthLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Cloth Width:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clothWidthEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="clothSideLenLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Cloth Side Len:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clothSideLenEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="gravityLabel">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Gravity:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="gravityEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="startSimulationButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Simulation</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="SimParametersBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>206</width>
        <height>89</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <widget class="QLabel" name="timeStepLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Time Step:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="timeStepEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="moonPiecesButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Moon Pieces:</string>
       </property>
      </widget>
      <widget class="QLabel" name="explosionMag">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Explosion Mag:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="moonPieceEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="explosionMagEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="clothDensityLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cloth Density: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clothDensityEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionReset_Everything"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Clear Scene</string>
   </property>
  </action>
  <action name="actionReset_Everything">
   <property name="text">
    <string>Reset Everything</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLPanel</class>
   <extends>QWidget</extends>
   <header>glpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
